FAQS.md

```diff
@@ -0,0 +1,54 @@
+**Markor - Frequently Asked Questions**
+
+**Markor - FAQ**
+
+This Frequently Asked Question will be continually updated to accommodate new information with respect to new updates and changes on the App and will be used inside the App to help users find answers to their questions.
+
+
+**What is Markor?**
+
+Markor is an Android application developed by Gregor Santner. It allows you to write, save and open files in markdown, plain and .txt formats. A note app developed for easy and improved note taking in markdown and .txt. 
+
+
+**What size is Markor and hope it isn't harmful to device?**
+
+Markor is a simple and lightweigh app that doesn't comsume much of your device space or RAM when installed, it is well diagonised for any harm it might cause your device and has been confirmed harm free.
```

Comment:
typos. and no, it is not professionally audited, so remove the last sentence. checks by bots or no references for security checks don't count.